testing/helpers/qunitExtensions.js

```diff
@@ -137,13 +137,30 @@
         });
     };
 
+    const beforeTestDoneCallbacks = [];
+
+    QUnit.beforeTestDone = function(callback) {
```

Comment:
Why you are not using arrow functions here?

Reply:
Arrow functions are not supported in IE:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions